app/helpers/request_info_helper.rb

```diff
@@ -0,0 +1,25 @@
+module RequestInfoHelper
+  def provision_tab_configuration(workflow)
+    prov_tab_labels = []
+    workflow.provisioning_tab_list.each do |dialog|
+      prov_tab_labels.push({:name => dialog[:name], :text => dialog[:description]})
+    end
```

Comment:
Minor, but this pattern is better written using `map`

```suggestion
    prov_tab_labels = workflow.provisioning_tab_list.map do |dialog|
      {:name => dialog[:name], :text => dialog[:description]}
    end
```